Change generated file headers to include GitHub link instead of version (#894)
giantswarm/devctl: CHANGELOG.md

## [Unreleased]

### Removed

> - Header of generated files do not contain the devctl version any more

Reviewer: ```suggestion
- devctl version is no longer added to generated file header
```
